README.md

```diff
@@ -89,21 +91,10 @@ Define your own mappings to swap the node under the cursor with the next or prev
 like function parameters or arguments.
 
 ```lua
-lua <<EOF
-require'nvim-treesitter.configs'.setup {
-  textobjects = {
-    swap = {
-      enable = true,
-      swap_next = {
-        ["<leader>a"] = "@parameter.inner",
-      },
-      swap_previous = {
-        ["<leader>A"] = "@parameter.inner",
-      },
-    },
-  },
-}
-EOF
+-- keymaps
+local swap = require("nvim-treesitter-textobjects.swap")
+vim.keymap.set("n", "<leader>a", swap.swap_next("@parameter.inner"))
+vim.keymap.set("n", "<leader>A", swap.swap_next("@parameter.outer"))
```

Comment:
It is not clear `swap.swap_next` is a keymap (dot-repeatable) rhs or the command/action itself that can programmatically called. I don't find it's consistent with `move.goto_next_start`, etc. There is even `swap_textobject` (a local function) that is not a keymap.  Would you think having some module name like `mapping` or `keymap` somewhere would be a good idea?

Reply:
I agree that a consistent interface would be preferred, but I don't have any strong opinions about how to implement it

Reply:
`make_dot_repeatable` returns a function, so the wrapping isn't needed